test/unit/spec/v2/view-builder/views/ChallengeWebauthnView_spec.js

```diff
@@ -223,4 +222,70 @@ describe('v2/view-builder/views/webauthn/ChallengeWebauthnView', function() {
       })
       .catch(done.fail);
   });
+
+  it('shows correct text when Can\'t verify? is clicked', function() {
+    jest.spyOn(webauthn, 'isNewApiAvailable').mockReturnValue(true);
+    jest.spyOn(BrowserFeatures, 'isSafari').mockReturnValue(false);
+    testContext.init();
+    expect(testContext.view.$('.idx-webauthn-verify-text').text()).toMatchInlineSnapshot(
+      '"You will be prompted to use a security key or biometric verification (Windows Hello, Touch ID, etc.). Follow the instructions to complete verification."'
+    );
+    expect(testContext.view.$('.js-cant-verify').length).toBe(1);
+    expect(testContext.view.$('.js-help-description').css('display')).toBe('none');
+    const cantVerifyLink = document.getElementsByClassName('link js-cant-verify');
+    cantVerifyLink[0].click();
+    expect(testContext.view.$('.js-help-description').css('display')).toBe('block');
+    expect(testContext.view.$('.js-help-description')[0].textContent).toMatchInlineSnapshot(
```

Comment:
nit: either just compare to a simple string`$('.js-help-description').first().text()).toEqual('Are you trying...');`
or
compare the inline snapshot to html `$('.js-help-description').first().html()).toMatchInlineSnapshot`